arch/risc-v/include/arch.h

```diff
@@ -62,6 +62,12 @@ uint32_t up_gethartid(void);
 
 #endif
 
+/* Provide the maximum amount of page table levels per MMU type */
+
+#ifdef CONFIG_ARCH_MMU_TYPE_SV39
+#define ARCH_PGT_MAX_LEVELS (3)
```

Comment:
Move this to riscv_mmu.h? It's seems only used in mmu driver.

Reply:
It will eventually be needed elsewhere too. I'm implementing `CONFIG_BUILD_KERNEL` for RISC-V / MPFS target.

Reply:
It normally doesn't need to expose so detailed info outside arch, how about we move the macro to mmu.h until you really need it?

Reply:
No problem, I can move it for now and present this later here.

However at some point the amount of page table levels will be needed by this file as well, as the group_addrenv_s structure will be presented here (and it will need the page table amount).

Reply:
Sure, let's wait you new patch.